decidim-participatory_processes/app/helpers/decidim/participatory_processes/application_helper.rb

```diff
@@ -13,7 +13,7 @@ module ApplicationHelper
       #
       # Returns an HTML safe String.
       def render_participatory_processes_hook(hook_name)
-        Decidim::ParticipatoryProcesses.view_hooks.render(hook_name, self)
+        Decidim::ParticipatoryProcesses.view_hooks.render(hook_name, deep_dup)
```

Comment:
Can you improve the docs to explain why we need to send the deep_copy? I know it's on the public `docs/` folder, but I think it's worth it having it here too 😄